lib/gems/pending/appliance_console/cli.rb

```diff
@@ -144,6 +154,7 @@ def run
       uninstall_ipa if uninstall_ipa?
       install_ipa if install_ipa?
       install_certs if certs?
+      set_date_time if date_time?
```

Comment:
This only allows you to configure time sync if you are also setting the date and time. I feel like this should be:

```ruby
set_date_time if date_time? || auto_sync?
```